### PR TITLE
Automatic deployment

### PR DESCRIPTION
**Pull request by mpetrunic**
when tagging master it should automatically publish package to npm registry

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+language: node_js
+
+node_js:
+  - lts/*
+
+cache: yarn
+
+before_deploy:
+  - yarn install
```
Comment (wemeetagain) [inline .travis.yml]: I think yarn install will be called by the install step, so it shouldn't be needed explicitly?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+language: node_js
+
+node_js:
+  - lts/*
+
+cache: yarn
+
+before_deploy:
```
Comment (wemeetagain) [inline .travis.yml]: Can we either add a noop `"test"` script in package.json or a noop `test` step here? travis currently erroring out

Comment (mpetrunic) [reply .travis.yml]: Yeah, I know. I think I need to add script: true

File: .travis.yml (hunk 1 above)
Comment (mpetrunic) [reply .travis.yml]: Not sure, probably not because I've overridden default deploy script

**Pull request merged**